general-concepts/dependencies/text.xml

```diff
@@ -608,6 +619,54 @@ RESTRICT="!test? ( test )"
 DEPEND="test? ( dev-util/foo )"
 </codesample>
 
+</body>
+</section>
+
+<section>
+<title>Circular Dependencies</title>
+<body>
+
+<p>
+Circular dependencies occur if one or more of package's (possibly indirect)
+dependencies depend on the package itself. This creates a dependency cycle where
+each of the packages must technically be installed before the other.
+For example, if package A depends on B, B depends on C and C depends on A, then
+the package manager cannot install A before C, and C before A.
+</p>
+
+<p>
+There are three kinds of circular dependencies:
+</p>
+
+<ol>
+  <li>
+    Weak circular dependencies that occur if only of the packages strictly need
```

Comment:
I think the word "one" is missing in this line.

Reply:
Fixed, thanks.